packages/components/src/Disclosure/docs/migration-notes.md

```diff
@@ -0,0 +1,7 @@
+Coming from Orbiter, you should be aware of the following changes:
```

Comment:
Disclosure was not a thing per se, we should note that and link to the the Accordion doc when available.

Reply:
yes the disclosure component exists https://wl-orbiter-storybook.netlify.app/?path=/docs/disclosure--docs
This is what I look at in Orbiter for disclosure and one of the reasons why we support a custom header.